src/dev-app/theme.scss

```diff
@@ -15,6 +15,9 @@
 @include angular-material-mdc-typography();
 @include mat-edit-typography(mat-typography-config());
 
+// MDC does not yet have an equivalent to `mat-core` to include this in.
+@include mat-mdc-strong-focus-indicators-base();
```

Comment:
@jelbourn @mmalerba MDC doesn't seem to have an equivalent of `mat-core`... so I'm not sure where to put this mixin. I'm fine with either (1) leaving it with what we have here or (2) moving the `position: relative` back inside the `mat-mdc-strong-focus-indicators` mixin for now.

Reply:
Spoke offline, included this mixin within a new `_mat-mdc-core` mixin that is included within `mat-core`.